README.md

```diff
@@ -584,55 +566,60 @@ meant to be able to change with it.
     end
     ```
 
-  * <a name="let-and-let!"></a>
-    Use `let` and `let!` blocks instead of assigning values to instance
-    variables in `before` blocks.
-    <sup>[[link](#let-and-let!)]</sup>
+  * <a name="let-blocks"></a>
+    Use `let` and `let!` for data that is used across several examples in an
+    example group. Use `let!` to define variables even if they are not
+    referenced in examples, e.g. when testing negative cases.
+    Do not overuse `let`s for primitive data.
+    <sup>[[link](#let-blocks)]</sup>
 
     ```ruby
     # bad
-    describe '#type_id' do
-      before do
-        @resource = FactoryBot.create(:device)
-        @type = Type.find(@resource.type_id)
-      end
+    it 'combines' do
+      date = Date(2010, 01, 01)
+      epoch = Epoch.new
+      expect(formatter.format(:regular, name, surname)).to eq('John Wayne')
```

Comment:
this doesn't define name and surname. Not sure where this example comes.

Reply:
Extremely sorry for committing the version that I thought was complete. My mistake.